README.md

```diff
@@ -61,8 +61,45 @@ $ oc adm taint nodes <NodeNames> node.ocs.openshift.io/storage=true:NoSchedule
 
 ### Installation
 
+
 The OCS operator can be installed into an OpenShift cluster using Operator Lifecycle Manager (OLM).
 
+Option 1: Using `make install`  
+
+If you have a development environment or private image and want to install the OCS operator, follow the steps below:
+
+- Label Worker Nodes:  
+  OCS Operator will install its components only on nodes labeled for OCS.
+
+  ```console
+  oc label nodes <NodeName1> cluster.ocs.openshift.io/openshift-storage=''
+  oc label nodes <NodeName2> cluster.ocs.openshift.io/openshift-storage=''
+  oc label nodes <NodeName3> cluster.ocs.openshift.io/openshift-storage=''
+  ```
+
+- Set Environment Variables:  
+  Define the required variables for your private image:
+
+  ```console
+  export REGISTRY_NAMESPACE=<your-registry-namespace>
+  export IMAGE_TAG=<your-image-tag>
+  ```
+
+- Run the following command:
+
+  ```console
+  make install
+  ```
+
+- Verify Installation:    
+  Once the make install process completes, verify the status of the ClusterServiceVersion (CSV):
+
+  ```console
+  oc get csv -n openshift-storage
+  ```
+
+Option 2: Using Pre-Built YAML  - [Table of Contents](#table-of-contents)
```

Comment:
[Table of Contents](#table-of-contents) should not be here right?

Reply:
No... only in the top README file, right?

Reply:
Yes please remove it from here.

Reply:
done